src/Application/Model/Ticket.php

```diff
@@ -617,16 +617,7 @@ public static function createMissingEncodingTickets($project) {
 			return $handle->fetch()['create_missing_encoding_tickets'];
 		}
 		
-		public static function createMissingEncodingTicket($metaTicket, $encodingProfile) {
```

Comment:
Something went wrong on this line. The (untouched) function is not named `getDependingEncodingTicketState` anymore, but `getDependeeTicketState`

Reply:
Oh, right.